sre-operational-fabric.md

```diff
@@ -75,6 +75,7 @@ the world.
 ### Responsibilities
 
 - Help drive the architecture, develop, and maintain observability of customer environments
+- Help drive the architecture, develop, and maintain configuration maangement of customer environments
```

Comment:
`maangement` -> `management`

Reply:
Should we add a bullet back for observability? Not sure about losing that concept.